workers-site/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "private": true,
-  "name": "worker",
+  "name": "{{ project-name }}",
   "version": "1.0.0",
   "description": "A template for kick starting a Cloudflare Workers project",
   "main": "index.js",
-  "author": "Ashley Lewis <ashleymichal@gmail.com>",
+  "author": "{{ author }}",
```

Comment:
@victoriabernard92 this would also throw. would you recommend omitting this as well?

Reply:
I thought this didn't for some reason. Probably yes just to be safe. Can also add later. If someone is truly publishing their package to NPM they will likely change more in the package.json anyway